add notes about using and configuring SwitchBot entities (#31775)
home-assistant/home-assistant.github.io: source/_integrations/switchbot.markdown

Please note, device names configured in the SwitchBot app are not transferred into Home Assistant.

> Some SwitchBot devices need to be configured within the app before controlling them from Home Assistant, such as calibrating the cover open/close limits or pairing two covers to move together.

Reviewer: ```suggestion
Some SwitchBot devices need to be configured within the app before being controlled by Home Assistant, such as calibrating the cover open/close limits or pairing two covers to move together.
```

{% include integrations/config_flow.md %}

> 
> 
> 

Reviewer: ```suggestion
```

{% include integrations/config_flow.md %}

> SwitchBot devices are added to Home Assistant according to their domain, such as Curtains under the cover domain. For more information how to use these in automations, see the buliding block integration that domain, such as [cover](/integrations/cover/).

Reviewer: This sentence feels odd and more like a base concept and not related to SwitchBot at all. IMHO, adding this isn't right, as it causes this to be added to each entity platform we have for each integration page.

Author: I'm curious what you think of this other PR #31789 I made to do address this idea in a more general/automated way. I just feel it's unclear for newer users if they end up on any specific integration page like Switchbot, there is no mention of how to automate with the entities it made.
Thanks for the feedback!

> Some SwitchBot devices need to be configured within the app before controlling them from Home Assistant, such as adjusting the cover open/close limits or pairing two covers to move together.

Reviewer: This is a prerequisite which should be documented before the config flow section.
